utils/micro-controllers.stanza

```diff
@@ -186,6 +130,91 @@ public defn MicroController (properties:Tuple<KeyValue>, exist:Tuple<String>, so
   else :
     MicroController(jsons[0])
 
+;============================================================
+;======================= MCU to JITX ========================
+;============================================================
+
+defmethod to-jitx (mcu: MicroController) -> Instantiable :
+  pcb-component my-microcontroller :
+    manufacturer = manufacturer(mcu)
+    mpn = mpn(mcu)
+
+    to-jitx-pin-properties(pin-properties(mcu))
+    val bundle-table = to-jitx-bundles(bundles(mcu))
+    to-jitx-supports(supports(mcu), bundle-table)
+
+    assign-landpattern $
+      switch(mfg-package(mcu)) :
+        ; "SO8N" : SO8N
+        ; "TSSOP14" : TSSOP14
+        ; "TSSOP20" : TSSOP20
+        "LQFP100" : LQFP100
+        "LQFP128" : LQFP128
+        "LQFP144" : LQFP144
+        "LQFP176" : LQFP176
+        "LQFP208" : LQFP208
+        ; "LQFP32" : LQFP32
+        "LQFP48" : LQFP48
+        ; "LQFP64" : LQFP64
+        ; "LQFP80" : LQFP80
+        ; "UFQFPN20" : UFQFPN20
+        "UFQFPN28" : UFQFPN28
+        "UFQFPN32" : UFQFPN32
+        "UFQFPN48" : UFQFPN48
+        ; "VFQFPN36" : VFQFPN36
+        ; "VFQFPN68" : VFQFPN68
+        "LFBGA100" : LFBGA100
+        "LFBGA144" : LFBGA144
+        "LFBGA354" : LFBGA354
+        ; "LFBGA448" : LFBGA448
+        ; "LGA86" : LGA86
+        "TFBGA100" : TFBGA100
+        "TFBGA216" : TFBGA216
+        "TFBGA225" : TFBGA225
+        ; "TFBGA240" : TFBGA240
+        ; "TFBGA257" : TFBGA257
+        ; "TFBGA361" : TFBGA361
+        "TFBGA64" : TFBGA64
+        "UFBGA100" : UFBGA100
+        "UFBGA121" : UFBGA121
+        "UFBGA129" : UFBGA129
+        "UFBGA132" : UFBGA132
+        "UFBGA144" : UFBGA144
+        "UFBGA169" : UFBGA169
+        ; "UFBGA176" : UFBGA176
+        "UFBGA64" : UFBGA64
+        "UFBGA73" : UFBGA73
+        ; "EWLCSP49" : EWLCSP49
+        ; "EWLCSP66" : EWLCSP66
+        "WLCSP100" : WLCSP100
+        "WLCSP104" : WLCSP104
+        "WLCSP115" : WLCSP115
+        "WLCSP132" : WLCSP132
+        "WLCSP143" : WLCSP143
+        "WLCSP144" : WLCSP144
+        "WLCSP156" : WLCSP156
+        "WLCSP168" : WLCSP168
+        "WLCSP18" : WLCSP18
+        "WLCSP180" : WLCSP180
+        "WLCSP20" : WLCSP20
+        "WLCSP25" : WLCSP25
+        "WLCSP36" : WLCSP36
+        "WLCSP49" : WLCSP49
+        "WLCSP52" : WLCSP52
+        ; "WLCSP59" : WLCSP59
+        "WLCSP63" : WLCSP63
+        "WLCSP64" : WLCSP64
+        "WLCSP66" : WLCSP66
+        "WLCSP72" : WLCSP72
+        "WLCSP81" : WLCSP81
+        "WLCSP90" : WLCSP90
+        else :
+          fatal("MCU package %_ is not supported." % [mfg-package(mcu)])
```

Comment:
We don't have a solution in place for preventing MCUs with packages that are not supported.
My idea is to always include a dbquery parameter that restricts `mfg-package`:
```
{"mfg-package" : {"$in" : ["WLCSP66", "WLCSP72", "WLCSP81", "WLCSP90", ...]} }
```
Thoughts?

Reply:
I thought you had checked that none of the missing packages have mcus with those in the mongodb. Can you check that? If there are some, we should remove them form the db until we get the landpatterns done. Are those landpatterns planned for the next sprint?

Reply:
For now those mcus are in the testing dbs anywhy, not in the prod. SO we have to put documents in the prod anyway, so we can filter some out in the process

Reply:
Yes, let's remove MCUs with unsupported `mfg-package` when moving entries to `app`.